Sources/ProjectDescription/Environment.swift

```diff
@@ -5,6 +5,16 @@ public struct Environment {
     public enum Value {
         case boolean(Bool)
         case string(String)
+
+        public func getString(default defaultString: String = "") -> String {
```

Comment:
I'd not set a default here and rather require that value be passed. Let me describe a use case and what I think would happen:

The user is using this to configure the bundle ID through an env. variable. The way they use this method in their `Project.swift` is like this:

```swift
let target = Target(bundleId: Environment.bundleId.getString())
```

They'll end up with a project that has an empty bundle ID set and they'll think it's a Tuist bug.

Reply:
Hi @pepibumur 

Totally agree happy to remove the default value